src/layout/Header.js

```diff
@@ -8,9 +8,13 @@ const Header = () => {
     return (
       <AppBar position="fixed" sx={{ zIndex: (theme) => theme.zIndex.drawer + 1 }}>
         <Toolbar>
-          <Typography variant="h6" noWrap component="div">
+        <Typography variant="h6" noWrap component="div">
             {`Locale Spec Reference (iLib v${ilib.getVersion()})`}
```

Comment:
Should we use loctool to extract strings and resbundle to load them as an example?

Reply:
If we plan to support multi-language for this, we could use loctool to extract and generate resources.
it uses enact's i18n plugin. so I think we can wrap $L and load  strings.json.